### DISCUSSION TITLE
When running future downloads, does "deduplicate" PREVENT download of duplicates

### DISCUSSION THREAD

**Discussion by mikevictorbravo** [shadowmoose/RedditDownloader#185] [Q&A] [answered]
or does the software just check for downloads that are duplicates and delete them from your folder?
Unclear how this actually works.
Thanks,

**Comment by shadowmoose** [comment, marked as answer]
Copies of the same URL will always be skipped, but otherwise the files themselves must be initially downloaded. After downloading, the file is checked against all previous downloads, and if a match is found, it removes whichever file is lower quality.
